source/extensions/http/header_formatters/preserve_case/config.h

```diff
@@ -38,7 +38,9 @@ class PreserveCaseFormatterFactoryConfig
     : public Envoy::Http::StatefulHeaderKeyFormatterFactoryConfig {
 public:
   // Envoy::Http::StatefulHeaderKeyFormatterFactoryConfig
-  std::string name() const override { return "preserve_case"; }
+  std::string name() const override {
+    return "envoy.http.stateful_header_formatters.preserve_case";
```

Comment:
Is this a deprecation or a breaking change?

Reply:
It's not breaking, both names are valid I believe.